programs/create-genesis-ee/map_objects.hpp

```diff
@@ -27,14 +30,26 @@ struct comment_header : public chainbase::object<comment_header_object_type, com
     id_type id;
     uint64_t hash;
     uint64_t offset;
-    op_num create_op;
-    op_num last_delete_op;
+    operation_number create_op;
+    operation_number last_delete_op;
     int64_t net_rshares;
     int64_t author_reward;
     int64_t benefactor_reward;
     int64_t curator_reward;
 };
 
+struct vote_header : public chainbase::object<vote_header_object_type, vote_header> {
+    template<typename Constructor, typename Allocator>
+    vote_header(Constructor &&c, chainbase::allocator<Allocator> a) {
+        c(*this);
+    }
+
+    id_type id;
+    uint64_t hash;
+    uint64_t voter_hash;
+    uint64_t offset;
```

Comment:
There is no reason to store offset of operation.
Result vote object is small:
- voter
- weight
- time 

That's all. 
We need `rshares` instead of `time` for correct sorting, but it can be done later.

Lets remove offset, as result it will remove the second round of reading from source dump  with votes.